src/main/java/ur/inf/lab2/pz/servicemanmanagement/controller/ManagerRegisterController.java

```diff
@@ -36,6 +37,8 @@ public void navigateToLogin(ActionEvent event) throws IOException {
     @FXML
```

Comment:
Posortowałbym te pola tej klasy, UserSerwis niech bedzie wstrzykiwany tak jak ViewManager przez Seter a te wszystkie pola z adnotacją FXML niech będą koło siebie. Metodke navigateToLogin też trzeba stąd zabrać (tak - to chyba ja ją tam tam wrzuciłem :-D)